src/pages/dashboard/components/account-info/account-info.component.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from "react";
```

Comment:
`components/account_info/AccountInfo.component.tsx ` can we please follow this convention?

Reply:
I would have from the start but I am working on other PRs with this convention.